File: src/resources/car.js
db = connect("localhost:27017/app17");
coll = db.car;
coll.insert({
    "make" : "Ford",
    "model" : "Ranger",
    "year" : Number(1981),
    "size" : "10.5",
    "licensePlate" : "FR46XG",
    "licenseState": "CA",
    "vin": Number(77754126),
    "odometer" : Number(123202),
    "currentInsurer" : "AIG",
    "purchasedYear": Number(1979),
    "ownerNameTitle": "June Smith",
    "isAccident": false});
coll.insert({
    "make" : "BMW",
    "model" : "Series 5",
    "year" : Number(1981),
    "size" : "12.5",
    "licensePlate" : "XX46DG",
    "licenseState": "CA",
    "vin": Number(123333326),
    "odometer" : Number(124402),
    "currentInsurer" : "AIG",
    "purchasedYear": Number(1977),
    "ownerNameTitle": "Joey Smitheres",
    "isAccident": false});
coll.insert({
    "make" : "Merceds",
    "model" : "CLS",
    "year" : Number(1990),
    "size" : "9.5",
    "licensePlate" : "JREKW4",
    "licenseState": "CA",
    "vin": Number(284628736487),
    "odometer" : Number(238498),
    "currentInsurer" : "AIG",
    "purchasedYear": Number(1986),
    "ownerNameTitle": "Rick Richards",
    "isAccident": false});

File: src/main/java/app17/rest/Car.java
package app17.rest;

public class Car {

    String id = null;
    String make, model, size, licensePlate, licenseState, currentInsurer, ownerNameTitle;
    Number year, odometer, vin, purchasedYear;
    Boolean isAccident;

    public Car(String make, String model, Number year, String size,
               String licensePlate, String licenseState, Number vin, Number odometer,
               String currentInsurer, Number purchasedYear, String ownerNameTitle,
               Boolean isAccident) {
        this.make = make;
        this.model = model;
        this.year = year;
        this.size = size;
        this.licensePlate = licensePlate;
        this.licenseState = licenseState;
        this.vin = vin;
        this.odometer = odometer;
        this.currentInsurer = currentInsurer;
        this.purchasedYear = purchasedYear;
        this.ownerNameTitle = ownerNameTitle;
        this.isAccident = isAccident;
    }

    public void setId(String id) {
        this.id = id;
    }
}
